.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,50 @@
+name: Update
+
+on:
+  push:
+    branches:
```

Comment:
I suppose we should actually only trigger for changes in the template folder? We could also consider keeping this as manually triggered for now.